src/Raven.Client/Documents/Operations/Replication/PullReplicationAsHub.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using Raven.Client.Documents.Replication;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Client.Documents.Operations.Replication
+{
+    public sealed class PullReplicationAsHub : ExternalReplication
```

Comment:
Does it have to be in Client API? Does it have to be public? What user can do with this class?

Reply:
IMO, we should merge this with `PullReplicationDefinition` and use only `PullReplicationAsHub`, similar to how we handle `PullReplicationAsSink`. In that case, we would need to include it in the Client API because operations will be created with `PullReplicationAsHub` instead of `PullReplicationDefinition` (for example, `PutPullReplicationAsHubOperation`). However, this change would require modifications in many places, so I preferred not to do it as part of this PR. What do you think? :)

Reply:
So internal for now.

Reply:
done